README.md

```diff
@@ -1,199 +1,256 @@
 Python for OpenShift - Docker images
-========================================
+====================================
 
 This repository contains sources of the images for building various versions
 of Python applications as reproducible Docker images using
-[source-to-image](https://github.com/openshift/source-to-image).
-User can choose between RHEL and CentOS based builder images.
+[Source-to-Image](https://github.com/openshift/source-to-image).
+User can choose between CentOS and RHEL based builder images.
 The resulting image can be run using [Docker](http://docker.io).
 
 
 Versions
----------------
-Python versions currently provided are:
-* python-3.3
+--------
 
-RHEL versions currently supported are:
-* RHEL7
+Python versions currently provided:
```

Comment:
This is accidentally bad diff. Not making breaking changes, just cleaning up formatting.